src/Controllers/SettingsController.php

```diff
@@ -21,11 +34,54 @@ class SettingsController extends BaseController
      * @param Response $response
      */
     public function __construct(
+        Authenticator $auth,
         Config $config,
+        LoggerInterface $log,
+        Redirector $redirector,
         Response $response
     ) {
-        $this->config = $config;
-        $this->response = $response;
+            $this->auth = $auth;
```

Comment:
Interesting formatting which looks odd when comparing it tot he rest of the file